<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="latexEditor">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="latexPreviwer">
      <property name="toolTip">
       <string/>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">div.maketitle {text-align:center;}
h2.titleHead{text-align:center;}
div.maketitle{ margin-bottom: 2em; }
</string>
      </property>
      <property name="documentTitle">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello_World">
    <property name="title">
     <string>账号</string>
    </property>
    <addaction name="separator"/>
    <addaction name="login"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="textbf"/>
    <addaction name="emph"/>
    <addaction name="underline"/>
    <addaction name="separator"/>
    <addaction name="quato"/>
    <addaction name="lineFormula"/>
    <addaction name="paragraphFormula"/>
    <addaction name="separator"/>
    <addaction name="newLine"/>
    <addaction name="newPage"/>
    <addaction name="label"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="getFile"/>
    <addaction name="openFile"/>
    <addaction name="uploadCloud"/>
    <addaction name="loadCloud"/>
    <addaction name="multiuser"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>段落</string>
    </property>
    <addaction name="section"/>
    <addaction name="subsection"/>
    <addaction name="subsubsection"/>
    <addaction name="separator"/>
    <addaction name="paragraph"/>
    <addaction name="footnote"/>
    <addaction name="tableofcontents"/>
    <addaction name="separator"/>
    <addaction name="makeCenter"/>
    <addaction name="flushleft"/>
    <addaction name="flushright"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menuHello_World"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="textbf">
   <property name="text">
    <string>粗体(B)</string>
   </property>
   <property name="toolTip">
    <string>粗体(B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="emph">
   <property name="text">
    <string>斜体(I)</string>
   </property>
   <property name="toolTip">
    <string>斜体(I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="underline">
   <property name="text">
    <string>下划线(U)</string>
   </property>
   <property name="toolTip">
    <string>下划线(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="login">
   <property name="text">
    <string>登陆\注册</string>
   </property>
   <property name="toolTip">
    <string>登陆注册</string>
   </property>
  </action>
  <action name="getFile">
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="toolTip">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="uploadCloud">
   <property name="text">
    <string>上传到云端</string>
   </property>
   <property name="toolTip">
    <string>上传到云端</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="toolTip">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="lineFormula">
   <property name="text">
    <string>行内公式</string>
   </property>
   <property name="toolTip">
    <string>行内公式</string>
   </property>
  </action>
  <action name="paragraphFormula">
   <property name="text">
    <string>行间公式</string>
   </property>
  </action>
  <action name="quato">
   <property name="text">
    <string>引用</string>
   </property>
   <property name="toolTip">
    <string>引用</string>
   </property>
  </action>
  <action name="newLine">
   <property name="text">
    <string>换行</string>
   </property>
   <property name="toolTip">
    <string>换行</string>
   </property>
  </action>
  <action name="section">
   <property name="text">
    <string>章节</string>
   </property>
   <property name="toolTip">
    <string>章节</string>
   </property>
  </action>
  <action name="subsection">
   <property name="text">
    <string>二级章节</string>
   </property>
   <property name="toolTip">
    <string>二级章节</string>
   </property>
  </action>
  <action name="subsubsection">
   <property name="text">
    <string>三级章节</string>
   </property>
   <property name="toolTip">
    <string>三级章节</string>
   </property>
  </action>
  <action name="paragraph">
   <property name="text">
    <string>段落</string>
   </property>
   <property name="toolTip">
    <string>段落</string>
   </property>
  </action>
  <action name="tableofcontents">
   <property name="text">
    <string>创建目录</string>
   </property>
   <property name="toolTip">
    <string>创建目录</string>
   </property>
  </action>
  <action name="newPage">
   <property name="text">
    <string>换页</string>
   </property>
   <property name="toolTip">
    <string>换页</string>
   </property>
  </action>
  <action name="label">
   <property name="text">
    <string>添加标签</string>
   </property>
   <property name="toolTip">
    <string>添加标签</string>
   </property>
  </action>
  <action name="makeCenter">
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>居中</string>
   </property>
  </action>
  <action name="footnote">
   <property name="text">
    <string>脚注</string>
   </property>
   <property name="toolTip">
    <string>脚注</string>
   </property>
  </action>
  <action name="flushleft">
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="toolTip">
    <string>左对齐</string>
   </property>
  </action>
  <action name="flushright">
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="toolTip">
    <string>右对齐</string>
   </property>
  </action>
  <action name="loadCloud">
   <property name="text">
    <string>从云端加载</string>
   </property>
   <property name="toolTip">
    <string>从云端加载</string>
   </property>
  </action>
  <action name="multiuser">
   <property name="text">
    <string>多人编辑</string>
   </property>
   <property name="toolTip">
    <string>多人编辑</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
